### DISCUSSION TITLE
Update LinuxGSM "Ungültige Zahl/Invalid Number"

### DISCUSSION THREAD

**Discussion by dr-d-bug** [GameServerManagers/LinuxGSM#3592] [Q&A] [answered]
After Updating Linux GSM with "./rustserver ul" i receive now often the error message "/home/rustserver/lgsm/functions/info_distro.sh: Zeile 39: printf: 20.04: Ungültige Zahl."
...translated...
"/home/rustserver/lgsm/functions/info_distro.sh: Line 39: printf: 20.04: Invalid Number."

**Comment by PlayMTL** [comment, marked as answer]
Should be the same issue as in https://github.com/GameServerManagers/LinuxGSM/issues/3587

A fix will be developed.
